sdk/src/types/block/output/rent.rs

```diff
@@ -140,3 +148,44 @@ impl<T: Rent, const N: usize> Rent for [T; N] {
         self.iter().map(|elem| elem.weighted_bytes(config)).sum()
     }
 }
+
+pub struct MinimumStorageDeposit {
+    config: RentStructure,
+    token_supply: u64,
+    builder: BasicOutputBuilder,
```

Comment:
Maybe call the whole struct `MinimumStorageDepositBasicOutput`? For other output types it's different
Or it should be possible to provide the type of output?

Reply:
Are we ever interested about non basic though? We only use this for returns

Reply:
From the name it sounds like a generic thing and not only for this specific case
If we don't want it generic, then I would make it pub(crate) and maybe add a comment above that it's used for storage deposit returns

Reply:
I'll just rename it